### ISSUE TITLE
new_sql_code_generation generates wrong ON CONFLICT

### ISSUE THREAD

**Issue by RalphBergmannKMB** [simolus3/moor#1704]
**Describe the bug**

A clear and concise description of what the bug is. Please briefly describe what behavior you expected, what behavior you saw, and how the bug can be reproduced.

drift file:
```sql
CREATE TABLE companies
(
    companyid INTEGER not null,
    companydb INTEGER not null,
    name      TEXT    not null,
    primary key (companyid, companydb)
) without rowid AS Company;

insertOrUpdateAllCompanies:
INSERT INTO companies(companyid, companydb, name)
SELECT json_extract(j.value, '$.companyid'),
       json_extract(j.value, '$.companydb'),
       json_extract(j.value, '$.name')
FROM json_each(:json) AS j
WHERE true
ON CONFLICT (companyid, companydb) DO UPDATE SET name = excluded.name;
```

Without `new_sql_code_generation` it generates:
```sql
'INSERT INTO companies(companyid, companydb, name) ... ON CONFLICT (companyid, companydb) DO UPDATE SET name = excluded.name',
```

With `new_sql_code_generation` it generates (missing parenthesis around the columns in `ON CONFLICT`):
```sql
'INSERT INTO companies (companyid, companydb, name) ... ON CONFLICT companyid, companydb DO UPDATE SET name = excluded.name',
```

**Comment by simolus3** [comment]
Thanks for the report! I've fixed the problem in 518fdce2e9fd3c3b4b4692aaa90233e2f4731c89, I'll push a hotfix to pub later today.

**Comment by simolus3** [comment]
Fixed in `sqlparser` release `0.20.1`.
